core/modules/node/node.entity.inc

```diff
@@ -156,6 +155,15 @@ class Node extends Entity {
    */
   public $in_preview;
 
+  /**
+   * Cached access values.
+   *
+   * @var array
+   *
+   * @see Node::access()
+   */
+  private $rights = array();
```

Comment:
I don't see `$this->rights` being used below, should this be removed?

Reply:
yes.